### PR TITLE
[16.0][ADD] purchase_order_line_receipt_status

### PR DESCRIPTION
**Pull request by imlopes**

### CODE REVIEW

File: purchase_order_line_receipt_status/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    @api.depends("order_line")
+    def _compute_receipt_status(self):
```
Comment (ivantodorovich) [inline purchase_order_line_receipt_status/models/purchase_order.py]: There's no need to completely overwrite [the core method](https://github.com/odoo/odoo/blob/94b5ae780432569e0a6e8367cb46b4a099823737/addons/purchase_stock/models/purchase.py#L65), is there?

IMO we can keep the order-level method as-is, and simply focus on adding the order line one.
The result, even if the implementation is different, should be the same.. right?

**Review by ivantodorovich — COMMENTED** [review]
I'd be cool to add some unit tests

File: purchase_order_line_receipt_status/models/purchase_order.py (hunk 1 above)
Comment (ivantodorovich) [reply purchase_order_line_receipt_status/models/purchase_order.py]: @imlopes ?

Comment (imlopes) [reply purchase_order_line_receipt_status/models/purchase_order.py]: Hmmm ! I think you're right.
I will remove this block.

Comment (ivantodorovich) [reply purchase_order_line_receipt_status/models/purchase_order.py]: Thanks!

**Review by ivantodorovich — APPROVED** [review]

**Review by Camille0907 — APPROVED** [review]
LGTM, thank you

**Pull request merged**